packages/chain-adapters/src/evm/EvmBaseAdapter.ts

```diff
@@ -220,25 +221,25 @@ export abstract class EvmBaseAdapter<T extends EvmChainId> implements IChainAdap
     })
   }
 
-  async buildSendTransaction(tx: BuildSendTxInput<T>): Promise<{
-    txToSign: SignTx<T>
-  }> {
+  // builds a SignTx<T> using serializable input data
+  async buildSignTx({
```

Comment:
:crazyeyes-1:

Reply:
Once this is ready, we should probably insert this as a PR under this one in the stack, and update consumptions in it since this is a breaking change for sends